src/Forms/TreeMultiselectField.php

```diff
@@ -90,9 +91,9 @@ public function getSchemaStateDefaults()
         foreach ($items as $item) {
             if ($item instanceof DataObject) {
                 $values[] = [
-                    'id' => $item->obj($this->getKeyField())->getValue(),
-                    'title' => $item->obj($this->getTitleField())->getValue(),
-                    'parentid' => $item->ParentID,
+                    'id'        => $item->obj($this->getKeyField())->getValue(),
```

Comment:
Could you revert changes where assoc array values have been indented, it's not a convention we follow elsewhere and creates noise in the git history

Reply:
Yeah my IDE settings sorry. :) I'll revert.

Reply:
These have all been done.